consent-ui/src/app/ais/entry-page/initiation/common/initial-consent/consent-account-access-selection.component.spec.ts

```diff
@@ -1,21 +1,52 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { ConsentAccountAccessSelectionComponent } from './consent-account-access-selection.component';
+import { Access, ConsentAccountAccessSelectionComponent } from './consent-account-access-selection.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+import { ActivatedRoute } from '@angular/router';
+import { of } from 'rxjs';
+import { SessionService } from '../../../../../common/session.service';
+import { AccountAccessLevel } from '../../../../common/dto/ais-consent';
+import { AuthConsentState } from '../../../../common/dto/auth-state';
+import { AccountsConsentReviewComponent } from '../../accounts/accounts-consent-review/accounts-consent-review.component';
+import { DedicatedAccessComponent } from '../dedicated-access/dedicated-access.component';
 
-describe('EntryPageAccountsComponent', () => {
+describe('ConsentAccountAccessSelectionComponent', () => {
   let component: ConsentAccountAccessSelectionComponent;
   let fixture: ComponentFixture<ConsentAccountAccessSelectionComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ConsentAccountAccessSelectionComponent ]
-    })
-    .compileComponents();
+      declarations: [ConsentAccountAccessSelectionComponent],
+      imports: [ReactiveFormsModule, RouterTestingModule],
+      providers: [
+        {
+          provide: ActivatedRoute,
+          useValue: {
+            parent: { parent: { params: of({ authId: 'AUTH-ID' }) } },
```

Comment:
Suggest to replace multiple occurences of 'AUTH-ID' with some constant

Reply:
done